### DISCUSSION TITLE
intelli j and Andrew studio memory optimization techniques technical

### DISCUSSION THREAD

**Discussion by john-peterson** [termux/termux-app#3362] [Q&A]
I am looking for purely technical comments preferably only commands without any comments I can understand myself what it does without any English which I don't like very much human tongues don't police me and TTS either excuse the homonym errors

It's absolutely tedious I have a horrible connection all the time and need to use IntelliJ engine

Please just tell me how you managed to run it with four or six gig ram and the ram outputs and the  Java language settings

I can only get by so far with ack. Without the IntelliJ I am lost. I need to implement a few simple things but I only have my phone all summer. Exactly 100 summers ago hey man just like me with tortured by allergies and neural implants anxieties hatched a marvelous idea in complete isolation all summer. I also need to be alone all summer with only my solar panel and my phone. But I don't want to torture myself unnecessarily. If there is even a thread of chance of running the IntelliJ engine on directly on my phone I would take that Jones

**Comment by agnostic-apollo** [comment]
Should be possible to get it running with `proot` if still not natively. Also don't expect support from IntelliJ.

https://youtrack.jetbrains.com/issue/IDEA-280178/IntelliJ-Idea-isnt-starting-on-Termuxs-XFCE4-through-its-openjdk-17-package.

https://www.reddit.com/r/termux/comments/r6984k/smooth_ide_experiance_debian_on_termux/

Related https://youtrack.jetbrains.com/issue/IDEA-316704/Crash-a-few-seconds-after-the-start-of-loading-the-project.
